markdown/dev/reference/api/svg/render/en.md

```diff
@@ -2,4 +2,28 @@
 title: Svg.render()
 ---
 
-FIXME: Write docs
+The `Svg.render()` method will render a drafted
+[Pattern](/reference/core/pattern) as SVG.
+
+## Signature
+
+```js
+string svg.render()
+```
+
+## Svg.render() example
+
+```
+import { Aaron } from "@freesewing/aaron"
+
+// Load some public test measurements from the FreeSewing backend
+const measurements = (
+  await (
+    await fetch("https://backend3.freesewing.org/curated-sets/1.json")
+  ).json()
+).measurements
+
+const pattern = new Aaron({ measurements })
+
+const svg = new Svg(pattern).render()
```

Comment:
This is mostly an internal method.
We expose the *proper* way to render a pattern via the `pattern.render()` method, like this:

```mjs
const pattern = new Aaron({ measurements })
const svg = pattern.render()
```

My concern is that documenting things this way will lead people who are looking to render a pattern to SVG to this page, and this is not how to do that.

Calling svg.render() will bypass all lifecycle hooks, and not do any layout. I doubt it's what anything would want.

I appreciate the completionist approach, but this might actually cause more confusion.

Reply:
As suggested, I removed the example and added the warning.